library/src/main/kotlin/me/saket/filesize/internals.kt

```diff
@@ -8,8 +8,7 @@ internal fun Long.toStringAsFixed(): String {
 }
 
 internal fun Double.toStringAsFixed(): String {
-  val string = this.toStringAsFixed(digits = 2)
-  return if (string.endsWith(".0")) string.substringBeforeLast(".") else string
+  return this.removeSuffix(".0")
```

Comment:
```suggestion
  return this.toStringAsFixed(digits = 2).removeSuffix(".0")
```

Reply:
Thanks. Editing on a smartphone leads to stupid errors like this one :sweat_smile: